deployment/docker_compose/docker-compose.dev.yml

```diff
@@ -17,6 +17,8 @@ services:
     ports:
       - "8080:8080"
     environment:
+      - INTERNAL_MODEL_VERSION=${INTERNAL_MODEL_VERSION:-openai-chat-completion}
```

Comment:
This is actually not correct. This should be `INTERNAL_MODEL` (or this needs to change in `model_configs.py`). Also this is missing for the background container (or it wont work for slack and other bots)

Reply:
Oh, great catch! You are most certainly right!

This PR was rushed out because I've been away for the last 2 days relocating to SF and someone needed it. Definitely embarrassing and in general we have stricter testing/reviews so these mistakes don't reach main.